core/src/test/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycleSpec.scala

```diff
@@ -446,15 +457,17 @@ class BpmBillingProjectLifecycleSpec extends AnyFlatSpec {
     }
   }
 
-  it should "create a protected data landing zone if requested" in {
+  it should "create a protected data landing zone and attach a protected-data policy to the billing profile if requested" in {
```

Comment:
Only functional test change. Others are just updates for the changed function signature